include/drives.h

```diff
@@ -40,9 +40,36 @@ void VFILE_Register(const char *name,
                     const uint32_t size,
                     const char *dir);
 
+class imageDisk; // forward declare
+
 class DriveManager {
 public:
-	static void AppendDisk(int drive, DOS_Drive* disk);
+	using filesystem_images_t = std::vector<std::unique_ptr<DOS_Drive>>;
+	using raw_images_t        = std::vector<std::unique_ptr<imageDisk>>;
+	struct DriveInfo {
+		filesystem_images_t disks = {};
+		uint16_t currentDisk      = 0;
```

Comment:
Shouldn't it be `current_disk` as per our conventions?

Reply:
:+1; will do a rename-only commit, so the logic changes can be easier to see (1:1 with names).